src/Infrastructure/TemplateMailerInterface.php

```diff
@@ -6,11 +6,14 @@
 
 use WMDE\EmailAddress\EmailAddress;
 
+/**
+ * @phpstan-type TemplateArguments array<string, mixed>
+ */
 interface TemplateMailerInterface {
 
 	/**
 	 * @param EmailAddress $recipient The recipient of the email to send
-	 * @param array $templateArguments Context parameters to use while rendering the template
+	 * @param TemplateArguments $templateArguments Context parameters to use while rendering the template
```

Comment:
```suggestion
	 * @param TemplateArguments[] $templateArguments Context parameters to use while rendering the template
```

Reply:
Thank you.

Reply:
Oh I was confused because as it is `TemplateArguments` is specified like a type. This should actually be something like `array<string,mixed>`